Packs/RDPCacheHunting/Playbooks/playbook-RDP_Bitmap_Cache_-_Detect_and_Hunt.yml

```diff
@@ -0,0 +1,1525 @@
+id: RDP Bitmap Cache - Detect and Hunt
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: RDP Bitmap Cache - Detect and Hunt
+description: This playbook automates the collection and forensic analysis of RDP sessions cache data by collecting the cache files and convert it to an image, extract readable text from the image and build IOC's from text and finally enrich any extracted indicators for further hunting
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 7e0b59dc-2b02-4284-8ac8-7bef3ca61007
+    type: start
+    task:
+      id: 7e0b59dc-2b02-4284-8ac8-7bef3ca61007
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "33"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -1365
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: c1f17bf2-68b5-4e89-8a68-2aa65f6719f2
+    type: regular
+    task:
+      id: c1f17bf2-68b5-4e89-8a68-2aa65f6719f2
+      version: -1
+      name: OCR
+      description: Extracts text from an image.
+      script: '|||image-ocr-extract-text'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      entryid:
+        complex:
+          root: File
+          filters:
+          - - operator: startWith
+              left:
+                value:
+                  simple: File.Name
+                iscontext: true
+              right:
+                value:
+                  simple: original
```

Comment:
We want to OCR both sharpened and greyscale images, the original is only for display.